src/retabled/core.cljc

```diff
@@ -54,9 +54,9 @@
   (fn [filterable-map]
     (every? some?
             (for [[k f] filter-map
-                  :let [field-filter-fn (cond 
+                 :let [field-filter-fn (cond
                                           (fn? f) f
-                                          (string? f) (partial re-find (re-pattern (str f))) ;; TODO right now ints treated as strings. Update this? 
+                                          (string? f) (partial re-find (re-pattern (str "(?i)"  f))) ;; TODO right now ints treated as strings. Update this? 
```

Comment:
Make this toggle-able: `ignore-case?` should be an optional specification that defaults to `true` (which is the case you have successfully here), but it should be an option to enable case sensitivity if desired.